.circleci/config.yml

```diff
@@ -497,11 +502,17 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/verdaccio_dist
+      - run:
+          name: Install AzCopy
+          command: |
+            wget -O azcopy.tar.gz https://aka.ms/downloadazcopy-v10-linux
```

Comment:
probably no longer need the docker image we for azure-cli? can probably just use the rsp-large or xlarge executor

Reply:
Tried that but we don't have the same permissions on the executor. Will stick with the existing image for now

Reply:
We can't include this in the docker image setup or a package.json so it's always there?

Reply:
It's a super quick download so I think its fine. I don't see any official images dedicated to this package.